stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/list/adapter/internal/ChannelListItemAdapter.kt

```diff
@@ -37,16 +37,16 @@ internal class ChannelListItemAdapter(
     }
 
     override fun onBindViewHolder(holder: BaseChannelListItemViewHolder, position: Int) {
-        bind(position, holder, FULL_CHANNEL_LIST_ITEM_PAYLOAD_DIFF)
+        bind(position, holder, fullChannelListItemPayloadDiff())
     }
 
     override fun onBindViewHolder(holder: BaseChannelListItemViewHolder, position: Int, payloads: MutableList<Any>) {
         val diff = (
             payloads
                 .filterIsInstance<ChannelListPayloadDiff>()
                 .takeIf { it.isNotEmpty() }
-                ?: listOf(FULL_CHANNEL_LIST_ITEM_PAYLOAD_DIFF)
-            ).fold(EMPTY_CHANNEL_LIST_ITEM_PAYLOAD_DIFF, ChannelListPayloadDiff::plus)
+                ?: listOf(fullChannelListItemPayloadDiff())
+            ).fold(emptyChannelListItemPayloadDiff(), ChannelListPayloadDiff::plus)
```

Comment:
Doesn't it decrease the performance of our `ChannelListView`???
Per every bind we will need to create, at least` one `emptyChannelListItemPayloadDiff`